zones/trollpasta.com

```diff
@@ -0,0 +1,31 @@
+$TTL 300
+$ORIGIN trollpasta.com.
+
+@		SOA ns1.miraheze.org. hostmaster.miraheze.org. (
+		20180829000001	; serial
```

Comment:
date needs changing.

Can be 20200416000001

Reply:
@paladox Thanks. Doing now. Wondered about that.

Reply:
Should be done now.